src/backend/tests/models/charge_types/test_type_analyzer_duii.py

```diff
@@ -0,0 +1,49 @@
+
+from datetime import datetime, timedelta
+
+from expungeservice.models.expungement_result import EligibilityStatus
+
+from tests.factories.charge_factory import ChargeFactory
+from expungeservice.models.disposition import Disposition
+
+
+def test_duii_acquitted():
+
+    last_week = (datetime.today() - timedelta(days=7)).strftime('%m/%d/%Y')
+    single_charge = ChargeFactory.build()
+
+    single_charge['statute'] = '813.010'
+    single_charge['disposition'] = Disposition(ruling='Acquitted', date=last_week)
```

Comment:
I forgot to mention that we probably should extract these common lines out

```python
def build_charge(statute, disposition_ruling):
    last_week = (datetime.today() - timedelta(days=7)).strftime('%m/%d/%Y')
    charge = ChargeFactory.build()
    charge['statute'] = statute
    charge['disposition'] = Disposition(ruling=disposition_ruling, date=last_week)
    return charge
```

and then use like

```
single_charge = build_charge(statute='813.010', disposition_ruling='Acquitted')
```

for each of these tests.